workshop/workshop_tags_224260_schinese.json

```diff
@@ -1 +1 @@
-{"language":"schinese","appid":224260,"category_0":"类别","tag_138673":"玩家皮肤","tag_138674":"武器皮肤","tag_138675":"模型包","tag_138676":"音效包","tag_138677":"HUD \/ UI","tag_138678":"脚本 \/ 资源包","tag_138755":"任务地图","tag_138756":"生存地图","tag_138757":"自定义地图","tag_138758":"其它","tag_332066":"游戏内容修改器"}
\ No newline at end of file
+{"language":"schinese","appid":224260,"category_0":"类别","tag_138673":"玩家皮肤","tag_138674":"武器皮肤","tag_138675":"模型包","tag_138676":"音效包","tag_138677":"游戏内显示 \/ 用户界面","tag_138678":"脚本 \/ 资源包","tag_138755":"任务地图","tag_138756":"生存地图","tag_138757":"自定义地图","tag_138758":"其它","tag_332066":"游戏内容修改器"}
```

Comment:
> HUD is a proper noun and "游戏内显示" can't express the meaning of it accurately.

because it is a proper noun, soit is translated as
this is the exact translation of HUD/UI
head up display/user interface
In fact, it shows the content and user interface in the game
Of course, as a proper term, it can be seen in most games or software. So most people know what this option is
If you insist 
I withdraw this line

Reply:
Better keep it as HUD.

Reply:
> Better keep it as HUD.

ofcourse